OpenOversight/app/templates/cop_face.html

```diff
@@ -139,3 +144,5 @@ <h4><small class="muted">{{ department.name }}</small></h4>
 
 </div>
 {% endblock %}
+
+{% block footer_class %}bottom-10{% endblock %}
```

Comment:
What does this extra class do?

Reply:
It bumps the footer down to just 10px instead of 140px from the bottom of the page to prevent it overlapping the explanation.